declarations/WebpackOptions.d.ts

```diff
@@ -2988,6 +2988,15 @@ export interface CssParserOptions {
 	 */
 	namedExports?: CssParserNamedExports;
 }
+/**
+ * Options for defer import.
+ */
+export interface DeferImportExperimentOptions {
+	/**
+	 * The execution of async modules (module that uses top-level-await or AsyncWebAssembly) cannot be deferred. This options controls how to handle them: "error", to throw an error when an async module is deferred; "ignore", to ignore the `webpackDefer` annotation when importing an async module, or one with async dependencies; "proposal", to eagerly evaluate the async subgraphs of a deferred module graph (this matches the behavior of the TC39 deferred imports proposal).
+	 */
+	asyncModule: "ignore" | "proposal" | "error";
```

Comment:
The proposal reached stage 2.7 (which is basically stage 3, with missing test262 tests) at the last TC39 meeting. Would you like me to remove this option and just use the proposal semantics?

Reply:
I think it's ok to remove the others since it is stage 2.7